docker-compose.yml

```diff
@@ -7,6 +7,16 @@ services:
       - 54325:5432
     env_file: docker/.env
 
+  api:
```

Comment:
Why not merge Web and API docker container ?

Reply:
It's to keep services separated

Reply:
I like very much the separation of services, but this could be a good way to achieve #42 for the frontend using the python ability to get and use env var :thinking:

Reply:
2 against 1, majority wins ^^
I'll include the api in the web service, just give me some time :p

Reply:
I think it could be done in another PR, there is already a lot of things in this one.